lib/ffmpegCommandService.js

```diff
@@ -515,28 +530,45 @@ class FfmpegCommandService {
         return {filter, maps};
     }
 
-    _createFilterForTextOverlay(textOverlay, i) {
-        const bashCharsToEscapeRegex = /!|'|"/g;
+
+    /**
+     * Scales a number in relation to how the base font size relates to the
+     * output height, which ensures font sizes are consistent across various
+     * output dimensions.
+     */
+    _calculateScalableFontSize(size, outputHeight) {
+        return (size / BASE_FONT_SIZE) * (outputHeight / BASE_FONT_SIZE);
+    }
+
+    _createFilterForTextOverlay(textOverlay, i, command) {
+        const bashCharsToEscapeRegex = /!|"/g;
 
         // @TODO: add handling for escaping special characters in bash.  Know it's at least ! that needs escaping.
-        const text = textOverlay.text.replace(bashCharsToEscapeRegex, "\\$&");
+        const text = textOverlay.text
+            .replace(bashCharsToEscapeRegex, "\\$&")
+            // No other replacement for single quote worked
+            .replace(/'/g, "\u2019");
```

Comment:
Fixes single quotes. No other replacement worked... likely due to some combo of escaping and the default wrapper being single quotes for text.